### ISSUE TITLE
Can't login to Modrinth Desktop App (with Modrinth Account) (Windows 11, Theseus v0.7.1, Theseus v0.6.3)

### ISSUE THREAD

**Issue by ghost** [modrinth/code#1163]
### Describe the bug

I can actually log in with my account data via the browser, but not via the desktop app. I get the following error messages every time:

An error occurred
Error: Failed to login with error Captcha Error. Try resubmitting the form.

An error occurred
invalid args `challenge` for command `login_pass`: command login_pass missing required key challenge

![modrinth wont let me login issue](https://github.com/modrinth/theseus/assets/130899981/43d1d411-d328-414e-9d80-942bfa473e0f)

The first time I click on the 'Login' button, I get the first error message, the second time I get the second message. The last error message then remains and I cannot log in.

What can be done? My account details are completely correct and I am connected to the Internet normally.

I have already tried restarting my PC, deleting my browser history, logging out of Modrinth within the browser and logging in and out of the Modrinth app with my Minecraft account.

It should be mentioned that this happens in the two versions I mentioned. (I didn't check if it would work on other versions instead.)

### Steps to reproduce

1. Start Modrinth Windows app (either of those versions: Theseus v0.7.1, Theseus v0.6.3)
2. Go to 'Settings' menu
3. Under 'General settings -> Manage account' click 'Sign In'
3. Enter Modrinth Account Data and see errors

### Expected behavior

To get signed into my Modrinth Account successfully.

### System information

Windows 11
Modrinth Desktop versions: Theseus v0.7.1, Theseus v0.6.3

**Comment by blryface** [comment]
You should use the discord for support, I think

**Comment by RokeJulianLockhart** [comment]
> https://github.com/modrinth/code/issues/1163#issue-2270944654

@ghost, I've the same:

![Image](https://github.com/user-attachments/assets/49ea9fd8-4f7c-4ca8-b857-0a3685c20aa0)

#### **Environment**

```bash
#!/usr/bin/env -S bash
flatpak install 'https://dl.flathub.org/repo/appstream/com.modrinth.ModrinthApp.flatpakref' --system && \
flatpak update 'app/com.modrinth.ModrinthApp/x86_64/stable' --commit=12863f64fd16c6b0e40bdec4d213243c26cf283c16316364b699161620fddb25
```

**Comment by brawaru** [comment]
It is correctly marked as completed, we've resolved CAPTCHA issues back then. The issue you are experiencing right now is caused by a different thing—we have changed CAPTCHA provider [on 17th of November](https://github.com/modrinth/code/commit/b188b3feb3953e10877dfc522b8fef5deb75d33e), but this change has not yet made it to the app.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/modrinth/code/issues/1163#issuecomment-2494137663

@brawaru, worth reporting [downstream](https://github.com/flathub/com.modrinth.ModrinthApp/issues/new), then?

**Comment by RokeJulianLockhart** [comment]
> https://github.com/modrinth/code/issues/1163#issuecomment-2494137663

@brawaru, worth reporting [downstream](https://github.com/flathub/com.modrinth.ModrinthApp/issues/new), then? (I presume that the `.RPM` doesn't experience this?)

**Comment by tale** [comment]
Is there any kind of update on this? It's now March and on my mac with the latest version the Captcha just flat out does not load.

**Comment by Prospector** [comment]
Are you on the latest version of Modrinth App? Should be 0.9.3

**Comment by tale** [comment]
![Image](https://github.com/user-attachments/assets/6d5a70de-c4a5-4ae7-8cfb-76f737ca712c)

Yes, and this is what I see. No matter what network I'm on that captcha never loads. Logging in with GitHub or any social just silently errors and does nothing.

**Comment by okseb** [comment]
Yeah I still get this issue yet I'm also still paying for the subscription.

**Comment by Neddoo** [comment]
Getting this as well, GitHub auth also doesn't work.
Wasn't really able to debug this issue myself.

**Comment by Gritania** [comment]
I too am having this same issue

**Comment by finnglink** [comment]
+1 on this issue. The Sign In button is greyed out, no captcha visible

Modrinth App 0.9.3
MacOS 14.6.1

![Image](https://github.com/user-attachments/assets/b3466ccf-59d7-461b-a59e-33ffb3f7e0e4)

**Comment by u6ma** [comment]
same for me

**Comment by Prospector** [comment]
Sign in has been overhauled
